src/components/previous-next/template.njk

```diff
@@ -0,0 +1,37 @@
+
+<nav class="govuk-c-previous-next
+  {%- if params.classes %} {{ params.classes }}{% endif %}" role="navigation" aria-label="Previous-next"{% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
```

Comment:
I don't think this is a good aria label for this component. We probably need to continue to call it 'Pagination' here?